contracts/sol6/Dao/KyberFeeHandler.sol

```diff
@@ -14,24 +14,25 @@ import "./DaoOperator.sol";
 /**
  * @title Kyber fee handler
  *
- * @dev Kyber fee Handler works tightly with contracts KyberNetwork and KyberDAO.
+ * @dev Kyber fee Handler works tightly with contracts KyberNetwork and KyberDao.
  *      Some events are moved to interface, easier for public uses
  * @dev Terminology:
- *          Epoch - DAO Voting campaign time frame.
- *              Kyber DAO voting campaigns have pre defined time period defined in number of blocks.
+ *          Epoch - Voting campaign time frame in the Dao.
+ *              Kyber Dao voting campaigns are in the scope of epochs.
  *          BRR - Burn / Reward / Rebate. Kyber network fee is used for 3 purposes:
  *              Burning KNC
- *              Reward addresse that stake KNC in KyberStaking contract. AKA - stakers
+ *              Reward an address that staked KNC in KyberStaking contract. AKA - stakers
  *              Rebate reserves for supporting trades.
  * @dev Code flow:
  *      1. Accumulating && claiming Fees. Per trade on KyberNetwork, it calls handleFees() function which
  *          internally accounts for network & platform fees from the trade. Fee distribution:
- *              rewards: accumulated per epoch. can be claimed by the DAO after epoch is concluded.
+ *              rewards: accumulated per epoch. can be claimed by the Dao after epoch is concluded.
  *              rebates: accumulated per rebate wallet, can be claimed any time.
- *              Burn: accumulated in the contract. Burned value and interval limited with safe check using sanity rate
+ *              Burn: accumulated in the contract. Burned value and interval limited with safe check using
+                    sanity rate.
  *              Platfrom fee: accumulated per platform wallet, can be claimed any time.
- *      2. Network Fee distribution. per epoch Kyber fee Handler reads current distribution from Kyber DAO.
- *          Expiry timestamp for data is set. when data expires. Fee handler reads new data from DAO.
+ *      2. Network Fee distribution: Per epoch fee Handler contract reads BRR distribution percentage 
```

Comment:
here too.

Reply:
Need to finalize the term we use for Dao as well.
KyberDAO or KyberDao or Kyber Dao or DAO or Dao